kafka-transport-binding.md

```diff
@@ -134,20 +137,20 @@ value of the Kafka message.
 #### 3.2.3. Metadata Headers
 
 All [CloudEvents][CE] attributes and
-[CloudEvent Attributes Extensions](primer.md#cloudevent-attribute-extensions)
-with exception of `data` MUST be individually mapped to and from the Header
+[CloudEvent Attributes Extensions](primer.md#cloudevent-attribute-extensions) 
+with exception of `data` MUST be individually mapped to and from the Header 
 fields in the Kafka message.
 
 ##### 3.2.3.1 Property Names
 
-CloudEvent attributes are prefixed with "cloudEvents_" for use in the
+CloudEvent attributes are prefixed with "ce_" for use in the
 [message-headers][Kafka-Message-Header] section.
 
 Examples:
 
-    * `eventTime` maps to `cloudEvents_eventTime`
-    * `eventID` maps to `cloudEvents_eventID`
-    * `cloudEventsVersion` maps to `cloudEvents_cloudEventsVersion`
+    * `time` maps to `ce_time`
+    * `id` maps to `ce_id`
+    * `specVersion` maps to `ce_specVersion`
```

Comment:
s/specVersion/specversion/

Reply:
yeah - there were quite a few out of date attributes